Cargo.toml

```diff
@@ -21,6 +21,9 @@ targets = [
   "x86_64-apple-darwin"
 ]
 
+[patch.crates-io]
+tao = { git = "https://github.com/wravery/tao", branch = "webview2-com" }
```

Comment:
Let's change the dependency directly for now. Otherwise tauri won't get the correct update of tao.

Reply:
OK, I included a commented out line to switch to the "next" branch of TAO after that's merged.